### ISSUE TITLE
Allow an array for accessibilityTraits

### ISSUE THREAD

**Issue by renaekathleen** [FormidableLabs/eslint-plugin-react-native-a11y#36]
As per the documentation here https://facebook.github.io/react-native/docs/accessibility.html#accessibilitytraits-ios an array, e.g. ["button", "disabled"] should be valid for the accessibilityTraits attribute, but only a string passes the ESLint rule.

**Comment by renaekathleen** [comment]
Nevermind, this is fixed by https://github.com/FormidableLabs/eslint-plugin-react-native-a11y/pull/31 I believe :D

**Comment by knitcodemonkey** [comment]
@renaekathleen Would you mind verifying that you can now validate arrays in v1.1.0?

**Comment by renaekathleen** [comment]
@knitcodemonkey Perfecto, thanks so much! 🎉

**Comment by draperunner** [comment]
@knitcodemonkey Seems like v1.0.0 is still the latest release on npm. Forgot an `npm publish`, or waiting for something? :)
